Sources/PointFree/PublicEpisodes/References.swift

```diff
@@ -0,0 +1,273 @@
+A few months after releasing our episode on [Contravariance](/episodes/ep14-contravariance) we decided to
+rename this fundamental operation. The new name is more friendly, has a long history in mathematics,
+and provides some nice intuitions when dealing with such a counterintuitive idea.
+""",
+    link: "https://www.pointfree.co/blog/posts/22-some-news-about-contramap",
+    publishedAt: Date(timeIntervalSince1970: 1540785600),
+    title: "Some news about contramap"
+  )
+
+  static let swiftNonEmpty = Episode.Reference(
+    author: "Brandon Williams & Stephen Celis",
+    blurb: """
+`NonEmpty` is one of our open source projects for expressing a type safe, compiler proven non-empty
+collection of values.
+""",
+    link: "https://github.com/pointfreeco/swift-nonempty",
+    publishedAt: Date(timeIntervalSince1970: 1532491200),
+    title: "NonEmpty"
+  )
+
+  static let swiftOverture = Episode.Reference(
+    author: "Brandon Williams & Stephen Celis",
+    blurb: """
+We open sourced the Overture library to give everyone access to functional compositions, even if you can't
+bring operators into your codebase.
+""",
+    link: "https://github.com/pointfreeco/swift-overture",
+    publishedAt: Date(timeIntervalSince1970: 1523246400),
+    title: "Swift Overture"
+  )
+
+  static let swiftTagged = Episode.Reference(
+    author: "Brandon Williams & Stephen Celis",
+    blurb: """
+`Tagged` is one of our open source projects for expressing a way to distinguish otherwise indistinguishable
+types at compile time.
+""",
+    link: "https://github.com/pointfreeco/swift-tagged",
+    publishedAt: Date(timeIntervalSince1970: 1523851200),
+    title: "Tagged"
+  )
+
+  static let swiftValidated = Episode.Reference(
+    author: "Brandon Williams & Stephen Celis",
+    blurb: """
+`Validated` is one of our open source projects that provides a `Result`-like type, which supports a `zip`
+operation. This means you can combine multiple validated values into a single one and accumulate all of
+their errors.
+""",
+    link: "https://github.com/pointfreeco/swift-validated",
+    publishedAt: Date(timeIntervalSince1970: 1534478400),
+    title: "Validated"
+  )
+
+  static let taggedSecondsAndMilliseconds = Episode.Reference(
+    author: "Brandon Williams",
+    blurb: """
+In this blog post we use the [`Tagged`](/episodes/ep12-tagged) type to provide a type safe way for
+interacting with seconds and milliseconds values. We are able to prove to ourselves that we do not misuse
+or mix up these values at compile time by using the tagged wrappers.
+""",
+    link: "https://www.pointfree.co/blog/posts/6-tagged-seconds-and-milliseconds",
+    publishedAt: Date(timeIntervalSince1970: 1531886400),
+    title: "Tagged Seconds and Milliseconds"
+  )
+
+  static let theoremsForFree = Episode.Reference(
+    author: "Philip Wadler",
+    blurb: """
+This famous paper describes the "theorems for free" result, in which if you write down a generic function
```

Comment:
Not sure what this line is trying to convey: "'theorems for free' result" in particular. Is there a better word we can use here other than "result"?